.github/workflows/main.yaml

```diff
@@ -0,0 +1,15 @@
+name: "Run Tests"
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.10']
```

Comment:
I limited this to python version 3.10 because I didn't want to mess with installing multiple versions on my machine.